source/xor.js

```diff
@@ -0,0 +1,26 @@
+import _curry2 from './internal/_curry2';
+
+/**
+ * Exclusive disjunction logical operation.
+ * Returns `true` if one of the arguments is truthy and the other is falsy.
+ * Otherwise, it returns `false`.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @sig a -> b -> Boolean
+ * @param {Any} a
+ * @param {Any} b
+ * @return {Boolean} true if one of the arguments is truthy and the other is falsy
+ * @see R.or, R.and
+ * @example
+ *
+ *      R.xor(true, true); //=> false
+ *      R.xor(true, false); //=> true
+ *      R.xor(false, true); //=> true
+ *      R.xor(false, false); //=> false
+ */
+var xor = _curry2(function xor(a, b) {
+  return Boolean(!a ^ !b);
```

Comment:
```suggestion
  return !a !== !b;
```

Reply:
There was a previous discussion about this here: https://github.com/ramda/ramda/pull/2646#discussion_r217921724

I'm ok with any implementation. Any thoughts @CrossEye @polytypic ?

Reply:
Yes, and sorry this got dropped.  Merged now.